bower.json

```diff
@@ -19,6 +19,7 @@
     "purescript-bifunctors": "master",
     "purescript-control": "master",
     "purescript-foldable-traversable": "master",
+    "purescript-generics-rep": "master",
```

Comment:
We'll be moving the `generics-rep` repo's code into `prelude`, so we can drop the dependency on `generics-rep`.